.gitignore

```diff
@@ -4,3 +4,4 @@ temp
 npm-debug.log
 packages/**/LICENSE
 .DS_Store
+!docs/recipes/build
```

Comment:
We've got an entry a few lines up that ignores `build` (in the context of the build output for a given module).

This is just to negate that so that I'm able to call the directory that hosts the recipes `build` and still check it in to git.